src/main.js

```diff
@@ -0,0 +1,20 @@
+function popin(text,isPermanent = true){
+    let div = document.getElementById("popInInfoDiv");
```

Comment:
This variable not going to change so let's go change to `const`
```suggestion
    const div = document.getElementById("popInInfoDiv");
```